src/utility/time_utils.h

```diff
@@ -1,8 +1,14 @@
 #ifndef TIME_UTILS_H
 #define TIME_UTILS_H
+#include <chrono>
+#include <cstdint>
+#include <iostream>
 #include <boost/date_time/posix_time/posix_time.hpp>
 namespace pt = boost::posix_time;
 
+/*
+ * Gets a timestamp as milliseconds since epoch.
+ */
 inline long long getCurrentMillis() {
```

Comment:
We need to fix getCurrentMillis to actually get time in milliseconds since epoch. Right now, it's getting milliseconds since midnight (Notice it's using time_of_day()).

Reply:
This was my code so it's up to you if you want to fix, just a note.

Reply:
Ah yeah I wasn't paying attention to the actual timestamps this method was returning. I'll fix it.